### DISCUSSION TITLE
Search Node Events : adding the option "Webobs code" in the "search for" field

### DISCUSSION THREAD

**Discussion by cyprien-griot** [IPGP/webobs#131] [Ideas]
Greetings all,
After some discussions about nodes' alias with the team here, we might have a new need.

Our technical team wanted to change some GNSS node's alias to unify them by adding the letter G at the end (so search on nodes can be handier). Unfortunately that would also change the name on the associated generated maps which would be a problem with official communication. 
So we came to the solution that adding a new option in the search field might resolve the problem as the webobs code of GNSS nodes is the same for all GNSS stations (well it differs depending of the network, but I don't think we can do better than this..)

Also, adding a help box specifying codes and the associated type of nodes would be a good idea :)

What do you think ?

Regards,
Cyprien

**Comment by beaudu** [comment]
The node alias should not be functional but only for display purpose (maps, ...). If you want to search nodes from GNSS networks, you should associate the nodes into a grid (e.g. a view), and use the combined search fields on grid name + node alias/name.
